ctt/model.py

```diff
@@ -19,13 +19,15 @@ class ReplicationResourceElement(ReplicationResourceOptions):
     source: ocm.Resource
     target: ocm.Resource
     component_id: ocm.ComponentIdentity
+    src_ocm_repo: ocm.OciOcmRepository
 
     @property
     def src_ref(self) -> oci.model.OciImageReference:
         if self.source.access.type is ocm.AccessType.OCI_REGISTRY:
             return oci.model.OciImageReference(self.source.access.imageReference)
         elif self.source.access.type is ocm.AccessType.RELATIVE_OCI_REFERENCE:
-            return oci.model.OciImageReference(self.source.access.reference, normalise=False)
+            full_ref = f'{self.src_ocm_repo.oci_ref.rstrip("/")}/{self.source.access.reference.lstrip("/")}' # noqa: E501
```

Comment:
might use ci.util.urljoin

also: since python3.12, it is no longer needed to use different quotation-style within expressions in fstrings :-)